index.bs

```diff
@@ -874,9 +875,12 @@ The <dfn method for=CookieStore>subscribeToChanges(|subscriptions|)</dfn> method
     1. [=list/For each=] |entry| in |subscriptions|, run these steps:
         1. Let |name| be |entry|'s {{CookieStoreGetOptions/name}} member.
         1. Let |url| be |entry|'s {{CookieStoreGetOptions/url}} member.
+        1. If |url| does not start with |registration|'s [=service worker registration/scope url=],
+            then reject |p| with a {{TypeError}} and abort these steps.
```

Comment:
Does this URL get parsed?  Are relative URLs evaluated against a base URL somewhere?  This needs to happen before comparing to the scope string.

Reply:
Added a snippet about that. (There's likely a ton of such things missing.)